components/Layout.vue

```diff
@@ -101,6 +111,7 @@ import { ref, onMounted, onUnmounted } from 'vue';
 const { path } = useRoute();
 const contentPath = path.endsWith('/') ? path.slice(0, -1) : path;
 const contentDirectory = await queryContent().where({ _path: contentPath }).findOne();
+const defaultImageSrc = contentDirectory.img;
 const imageSrc = contentDirectory.img;
```

Comment:
We can specify the default image here instead of getting it from the content page.

```
const imageSrc = contentDirectory.img || '/background.png';
```

This will make it so we don't need to add img front matter to every page, and will remove the other conditionals you have added to this file.

Reply:
Thanks